randovania/games/cave_story/gui/dialog/cs_cosmetic_patches_dialog.py

```diff
@@ -0,0 +1,74 @@
+import dataclasses
+import logging
+from functools import partial
+from PySide2.QtGui import QPixmap
+
+from PySide2.QtWidgets import QWidget
+from randovania.games.cave_story.layout.cs_cosmetic_patches import CSCosmeticPatches, MusicRandoType, MyChar
+from randovania.gui.dialog.base_cosmetic_patches_dialog import BaseCosmeticPatchesDialog
+from randovania.gui.generated.cs_cosmetic_patches_dialog_ui import Ui_CSCosmeticPatchesDialog
+from randovania.gui.lib.common_qt_lib import set_combo_with_value
+
+
+class CSCosmeticPatchesDialog(BaseCosmeticPatchesDialog, Ui_CSCosmeticPatchesDialog):
+    _cosmetic_patches: CSCosmeticPatches
+
+    def __init__(self, parent: QWidget, current: CSCosmeticPatches):
+        super().__init__(parent)
+        self.setupUi(self)
+        self._cosmetic_patches = current
+
+        for i in range(4):
```

Comment:
```python
for i, value in enumerate(MusicRandoType)
```
You can use `enum_lib.iterate_enum` if type hints goes crazy, like PyCharm does.

Reply:
:+1: